control_panel_api/elasticsearch.py

```diff
@@ -0,0 +1,26 @@
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Q, Search
+from elasticsearch_dsl.query import Range
+
+
+class ElasticSearch(object):
+    def __init__(self, connection):
+        self.client = Elasticsearch(connection)
+
+    def bucket_hits_aggregation(self, index, bucket_name, day_range=None):
```

Comment:
This method could be a function - just move the `client = Elasticsearch(connection)` in here and you don't use `self` for anything

Reply:
You could move the `index` param last and default to `settings.ELASTICSEARCH_INDEX_S3LOGS` so that the client code doesn't need to pass it unless it wants to change the default behaviour.